README.md

```diff
@@ -24,61 +24,88 @@ Pip will handle all dependencies automatically and you will always install the l
+
+Using the Qiskit-provided Sampler and Estimator will not take you very far. The power of quantum computing cannot be simulated 
+on classical computers and you need to use real quantum hardware to scale to larger quantum circuits. However, running a quantum 
+circuit on hardware requires rewriting them to the basis gates and connectivity of the quantum hardware.
+The tool that does this is the [transpiler](https://qiskit.org/documentation/tutorials/circuits_advanced/04_transpiler_passes_and_passmanager.html) 
+and Qiskit includes transpiler passes for synthesis, optimization, mapping, and scheduling. However, it also includes a
+default compiler which works very well in most examples. The following code will map the example circuit to the `basis_gates = ['cz', 'sx', 'rz']` and a linear chain of qubits with the `coupling_map =[[0, 1], [1, 2]]`.
 
 ```python
-{'00': 513, '11': 511}
+from qiskit import transpile
+qc_ibm = transpile(qc_example, basis_gates = ['cz', 'sx', 'rz'], coupling_map =[[0, 1], [1, 2]] , optimization_level=3)
 ```
 
-For further examples of using Qiskit you can look at the example scripts in **examples/python**. You can start with
-[using_qiskit_terra_level_0.py](examples/python/using_qiskit_terra_level_0.py) and working up in the levels. Also
-you can refer to the tutorials in the documentation here:
+For further examples of using Qiskit, look at the example scripts in **examples/python** and the tutorials 
+in the documentation here:
 
 https://qiskit.org/documentation/tutorials.html
 
 
-### Executing your code on a real quantum chip
+### Executing your code on real quantum hardware
+
+Qiskit provides an abstraction layer that lets users run quantum circuits on hardware from any vendor that provides a compatible interface. 
+The default way to use Qiskit is to have a runtime environment that provides optimal implementations of `sampler` and `estimator` for a given hardware. This runtime may involve using pre- and post-processing, such as optimized transpiler passes with error suppression, error mitigation, and, eventually, error correction built in. The runtime must provide a promise to the user that these primitives functions exist.
+
+* https://github.com/Qiskit/qiskit-ibm-runtime
 
-You can also use Qiskit to execute your code on a **real quantum processor**.
-Qiskit provides an abstraction layer that lets users run quantum circuits on hardware from any
-vendor that provides an interface to their systems through Qiskit. Using these ``providers`` you can run any Qiskit code against
-real quantum computers. Some examples of published provider packages for running on real hardware are:
+As Qiskit transitions to the runtime environment, some hardware is only supported with the ``providers`` interface; however, each provider may perform different types of pre- and post-processing and return outcomes that are vendor-defined:
 
 * https://github.com/Qiskit/qiskit-ibmq-provider
 * https://github.com/Qiskit-Partners/qiskit-ionq
```

Comment:
the ionq and aqt providers have been moved to qiskit community so we should probably update their links here:

https://github.com/qiskit-community/qiskit-ionq
https://github.com/qiskit-community/qiskit-aqt-provider

Reply:
https://github.com/Qiskit/qiskit-ibmq-provider is also deprecated (archived).
should it be https://github.com/Qiskit/qiskit-ibm-provider ?